data/language/en-GB.txt

```diff
@@ -4558,6 +4558,10 @@ STR_6248    :{WINDOW_COLOUR_1}Do you want to refurbish {STRINGID} for {BLACK}{CU
 STR_6249    :{WINDOW_COLOUR_1}Do you want to refurbish {STRINGID}?
 STR_6250    :{WINDOW_COLOUR_1}Are you sure you want to completely demolish {STRINGID} and gain {BLACK}{CURRENCY}?
 STR_6251    :Ride is not empty yet
+STR_6252    :Twitch API URL
+STR_6253    :URL of Twitch integration API
```

Comment:
While testing, I noticed this string is used for the input window as well as a tooltip. For tooltips, it should be prefixed with {SMALLFONT}.

Reply:
If the tooltip says the same as the button, it seems rather pointless. Unless the tooltip provides more information, I think it should be left out.